work/work.go

```diff
@@ -34,16 +34,43 @@ import (
 	"github.com/klaytn/klaytn/networks/p2p"
 	"github.com/klaytn/klaytn/params"
 	"github.com/klaytn/klaytn/storage/database"
+	"math/big"
 	"sync/atomic"
 )
 
 var logger = log.NewModuleLogger(log.Work)
 
+//go:generate mockgen -destination=work/mocks/txpool_mock.go -package=mocks github.com/klaytn/klaytn/work TxPool
+// TxPool is an interface of blockchain.TxPool used by ProtocolManager.
+type TxPool interface {
```

Comment:
Minor comment:
`package work` is for block creation and mining. Is it okay to add `TxPool` in this package?

Reply:
Could you please explain in detail? What can be different if `TxPool` interface is declared in another package?

Reply:
Assuming that you are looking for where the TxPool interface is defined, I will find the blockchain package first and then search other packages. In this case, `package work` is unlikely to be on a high priority to search the TxPool.

Alternatively, we can modify the description of the `package work`.